UndertaleModLib/Models/UndertaleEmbeddedTexture.cs

```diff
@@ -35,10 +35,16 @@ public class UndertaleEmbeddedTexture : UndertaleNamedResource, IDisposable
     public uint GeneratedMips { get; set; }
 
     /// <summary>
-    /// TODO: something. <br/>
+    /// The size of the texture data in the embedded image. <br/>
     /// GameMaker: Studio 2 only.
     /// </summary>
-    public uint TextureBlockSize { get; set; }
+    private uint _TextureBlockSize { get; set; }
```

Comment:
Is this an UMT-only attribute, or is this actually saved into the data.win?

Reply:
It's in the data file so definitely not UMT-only, I think the variable is no longer used in UML though

Reply:
Cool, just wanted to make sure.